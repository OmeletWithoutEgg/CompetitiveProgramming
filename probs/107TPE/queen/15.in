38
-119761315 5654060
87980206 -97548716
106421071 -55733673
57889768 -108676639
-112235675 -21618406
-99882825 44688924
-117623448 -9715376
-32931473 38582684
-83178931 -43393689
38596418 -105377009
-119718900 7013405
73839834 -11795411
105959464 -51303810
-21890652 -9635866
-74362817 54563033
-62382056 54225820
105899590 -66477625
96163619 -28240794
-34317128 40032997
48982352 -108125979
-66995761 -45148522
10097736 -40094494
83284944 -16469622
8604688 -44371328
-102695517 -32853247
101372590 -80769404
-116512390 -12975431
36841424 -104665169
15416794 -87181499
-97714437 46236001
-117088440 -11377131
106474227 -60557043
6673455 -62749098
103894215 -42738568
-118190789 -7688854
-117610652 19306583
-106237702 38980319
97248316 -29707805
