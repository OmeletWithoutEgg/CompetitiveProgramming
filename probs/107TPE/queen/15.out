-6270791.27 -30767451.25
