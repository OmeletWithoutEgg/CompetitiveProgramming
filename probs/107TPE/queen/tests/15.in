658
-115918466 59740952
126045866 -74376132
62228186 -81616331
88870699 -89907667
136417142 -21794471
56959223 -77604505
-110312115 52998594
-38346548 122417441
-69757938 137957819
59689986 -12079
113512606 -83945143
-80007771 137944434
-104160908 47730095
-27638202 71659268
-26464194 100939252
88153573 10055708
-108577259 125096833
-63119725 39602242
-70293643 38419620
67804946 4977437
-100378002 131187539
94060999 -89736165
-118399577 112722417
95948400 9705997
-48624993 130804787
87666256 10037109
52845638 -73578614
48586554 -68202007
137532840 -24977284
124930772 -75511733
134809580 -18107132
-123680901 77612285
-124816255 87808886
-99877533 44947724
73479849 7339095
85139323 -89697347
139092316 -48397651
139551597 -34773233
-123495919 76793015
70691713 -86114340
123889359 -76509840
67439236 4796272
-48587147 130781488
-32320813 61870677
-31112629 112501323
-25559699 79633743
118022176 1368478
-123644805 77447273
41335740 -52147013
96082359 9689260
-112735288 120807021
-124309986 81105820
-71388682 38332244
40167790 -45830278
-41071203 51320603
52321353 -6839617
-68086662 137759343
45042523 -62171059
-83280036 137493231
51208413 -71687502
139730567 -42852662
-65683765 137373107
-41904882 125854027
75651338 -87868037
64756913 -83183233
41429445 -52512634
68152601 5146232
133725442 -63833874
93630295 9937744
-47725735 46190534
-121172217 69472117
-62774575 136742423
-118150199 63268330
-103686527 47390468
-54406881 42570340
52085033 -7109449
92657054 -89835886
72880884 7127756
136603406 -22280926
-36716819 55836931
-118797424 112001092
41120966 -51261726
44141790 -19574806
131723870 -67189339
41649154 -53328315
-27988334 105735688
-48796030 45519520
53422050 -74201002
40109421 -45317735
-33914083 59459029
-44844448 128234407
-66923888 38841597
-111518853 122170955
-118606466 112350823
40029996 -35304714
-94068915 134359916
95228662 9789640
89701935 10083248
138043185 -53115866
135641183 -19913370
86764578 -89823369
41342599 -27659068
48837343 -11269098
-25604738 97049933
118002658 -81215045
-85833497 39443037
115223984 3147023
111232203 5265041
-39428554 52893211
77183225 -88294214
40283601 -46732615
129782171 -69962795
124578260 -3977171
-117776589 113798707
-108446842 51213236
-27871667 71008372
110297675 -85529508
-122821416 102202453
117852623 -81317043
-30363030 111100411
135883964 -20479013
139560157 -44976543
-76930714 38259170
138590897 -28912447
107182743 6970752
-27541734 104492227
139814695 -40380288
-86956234 39710275
-25046091 83443034
-28747124 68785574
76636326 8314843
63890388 2836379
81465032 -89214157
54902420 -4125797
127773302 -72463622
40226660 -46305174
-121806863 105303613
109930515 5859359
-119621336 66018741
-49220808 45263505
-25750592 97826964
-117089942 114917966
42556807 -56241105
125181280 -75262860
-124688957 91803070
130019259 -10188419
-79300482 138013170
-34080105 117204716
-54044455 42734142
66674259 -84238406
-124669507 84365187
68033038 5088556
-25019677 83727087
-117999989 113419811
44965508 -62015434
-77239108 138155854
133545233 -15671455
113868328 3918596
-111568842 122116846
52091472 -7102045
-26220388 76455183
-53779826 42855946
115419719 3030940
85406455 -89721724
96637802 9615941
86390354 9965834
-38611009 122697275
137472600 -24786222
139744186 -37222079
-24919981 91407890
-118442507 112645919
-122151326 104324969
-115214444 117678375
-121474912 70238881
39978552 -35898581
129608376 -70192584
87990696 -89883260
-83845949 137392700
112422090 4681619
-25014217 83788100
-118567830 64008237
106749918 -86962034
-28857314 68526329
137991270 -26529136
-86831188 136749868
-105673780 127558058
-112966835 120535692
139728957 -36953342
135424222 -60409720
-82644488 38830864
-60230610 136039305
-78490089 138079478
-110188023 52873961
-27999550 70663385
59377265 -79583129
132345700 -13624103
59338781 -79553568
-122927085 74594439
96342897 9655657
132384524 -13687005
-26248201 76340830
-77957963 138115813
138633274 -50731248
75271018 -87754041
118814629 815781
39865916 -37701962
-115671385 117041457
58667853 -804724
-36496049 56098537
-31314709 112861465
71626637 6657154
135897583 -59321915
-27872098 105421824
54225768 -4798600
45124295 -62334827
-93742720 41934372
-24902647 91124410
-34503120 58638870
40518762 -48265216
116746373 -82045000
-27988656 105736549
-70079435 38439547
61335968 -81012114
51015591 -71451723
-122923958 101845622
46879309 -65536300
-25016134 83766587
39828682 -41004577
-119393401 110864560
-38070251 54308607
-41285375 51125836
66488615 -84140976
133399763 -64422317
-41858926 125813793
-35591349 119218988
93350680 -89791583
-35330950 118886659
-25439679 80353772
42648899 -23327922
75843440 -87924369
139815192 -39510070
138661088 -50604924
113653737 -83868896
117501844 -81552431
42413215 -55819322
-98313997 44079111
-113628340 119738308
108559866 6437443
-73630544 38228611
85080514 9858547
-40777889 51591053
41801470 -25969440
-26932962 102603139
123859991 -3296076
-114163457 57361193
39888725 -42596717
55659038 -76430237
83869947 9728465
-117879913 62804610
-27648126 104798009
-102346466 129953918
124860906 -4252720
-62070689 39866723
44660582 -61385480
111370469 5199246
-27374481 103998108
-99708762 131578608
43427256 -58571541
46548150 -14860281
51492281 -72029356
-117910024 113573314
-26129234 99590526
120475113 -418891
-66412075 137502867
-70106597 38436969
-33051847 115702631
40152114 -45697155
40676980 -30682243
90095212 10082606
-124366419 81510455
-25962927 77575356
122186311 -78024593
106064281 -87203201
-25310452 81213611
-124109202 79825970
-53521963 42976440
55058583 -3974120
-25145876 82496996
-33650440 59837756
-32702719 115164677
117321325 1838662
136478479 -57880245
125177221 -4566319
51285035 -71780387
-35785068 119462518
-43417856 127125141
44965861 -62016151
-29284197 67558231
-49426565 131287473
-61251665 40090117
-46376130 47091941
-27340486 103895554
111331535 -85051064
-29559132 66962597
-61576705 39999671
-61880757 136511822
-117834947 62728559
-37278100 121241217
-72619848 38262848
-123224311 100737265
62337086 1854936
-122308609 72573776
139678600 -36201066
42455740 -55945524
42290121 -55447574
-96553966 133242762
-117568812 114144408
-108141697 50938183
-29855513 66342526
-112417546 121173070
86674413 9984534
-26918932 102556366
-35804207 56942610
-27161981 103344392
136732928 -22628440
-51221085 132296143
83231709 -89481082
139574971 -44828737
-65295116 39129721
116578285 -82151976
-25741930 97782640
139622295 -35510135
48914959 -11158390
-105524306 127674828
-122609875 73520049
63995821 2900138
-98923658 132019903
109435404 6073729
-24836351 86856292
105078047 7697409
114628480 -83326097
115563803 -82777956
-96011992 133500386
57079634 -77709146
85591321 -89737749
-49543458 131356168
58338071 -1069653
122067036 -78125586
-117215205 61710387
-55444243 42120461
127942400 -72265375
110311235 5690178
-124733900 91111578
-47591678 130151759
-24861301 86131675
-61287607 40079999
42668276 -56560311
-42332917 50204961
-89896530 135886671
42534494 -23656874
-32608519 61412121
52899518 -73637696
55942228 -3140127
39895608 -37111236
117939051 1425145
105176314 7665801
102965253 8323606
133406712 -15423285
-109457887 124271012
89399551 10081639
131452506 -12231394
-123005332 74873918
-55001952 134120155
-58591083 135508172
102486675 8451504
104014892 8025173
39853237 -41823947
-65373067 39114662
-87486588 136582934
-32857764 61023575
121557391 -1283244
72040536 -86649936
119165626 -80396850
80880316 9278283
-123465487 76664087
-36605155 120460224
132082928 -66629504
92640835 10003568
84403722 -89622737
138918548 -49351807
78527067 -88625355
54600242 -75410152
45776305 -16243198
-91399093 41043926
-116556292 115744616
69614725 -85653633
136575736 -57625546
-124157008 80111890
-99924364 131454139
-64554501 39279220
86724781 9987680
112115150 -84669112
-55071933 42278721
-104481251 128464798
-75809514 138204677
-120708226 68363111
-116360316 60389563
137009804 -56434022
65712413 3889503
-41085618 125121598
-47320936 46454279
-51605861 132499962
-110255669 52941794
-124677235 84466613
106261394 -87135014
-114493103 57786262
126802030 -73563254
52562426 -73264908
97581779 9476759
115008571 -83106629
-124664745 84304005
129361903 -9319424
-121217269 69583929
-100795378 130936527
-81350568 137785916
86664117 -89817124
-77633544 138135169
-82668311 38834646
-56150142 134598935
52252231 -72915102
-36875333 120777704
78143333 -88534817
83058382 9624507
-40121911 124217126
-34435614 117697911
-106616987 49629305
84337074 -89615434
-104566098 128402129
129031030 -8896503
-33087748 115757102
-24821456 88810769
96329533 9657415
-63255259 136859195
91764225 -89878682
139673651 -36135241
-111358122 122343856
41192715 -51565398
48734500 -11416778
-47776065 130270891
108874574 -86142185
-88924842 40245831
128692312 -8473051
-121709695 105568461
-109774103 123964530
118738972 869537
68395789 -85095579
-80351931 38521709
115316836 -82925345
39851547 -38054092
58141849 -78603758
-35547866 119163895
53547621 -5499422
-41647502 125627405
113905270 3898307
84045405 -89582407
-114238091 118972414
75937969 8118543
-79065495 138033763
50799855 -8648833
-62556876 39741144
39828050 -38858112
-28075547 70461979
-30354453 65345276
-26900273 102493900
-105452266 48698240
44037858 -19809794
-121664867 105689112
71189024 6483858
-73199450 38240711
-122737858 102486080
103413619 -88032394
-121236988 106795906
-124213668 80464792
-31908044 113880523
-59524987 135818363
94977765 9816316
130391024 -69135380
94345147 -89711142
-120277009 109034441
-84138973 137338004
-75377607 38217643
-76736675 138177680
-58368316 135431151
-94436385 42224130
-32966890 115573109
70665623 6270291
56025442 -3063653
-120700903 68346192
-124527671 93593979
-82900183 137556954
-46170076 129193800
-113772343 119560187
-25822508 98187049
-53439470 43015366
44797916 -61671988
-29174934 67800799
99433540 -88983096
128838838 -8655077
125715913 -5113503
-123611183 77295923
90048285 10082845
106461763 7231516
42129627 -54947563
62477704 1947105
-25440559 80348246
47997590 -12509504
84038716 -89581630
-25384090 80711287
123892048 -76507336
-87452125 136591947
108452224 -86314061
126212257 -5632892
-48147457 45921633
-86333661 39558707
127402319 -6941900
78065520 8682829
-43618511 127286215
-123912621 78743056
83094980 -89462725
-109539884 124192057
40221391 -46264135
125836578 -5238402
64155318 2995920
138868415 -30224176
139254125 -32436292
139513145 -34414102
123978236 -76426883
-47724087 46191597
86033718 -89773294
59749984 33147
74095849 -87380820
-29351605 67410281
109093427 -86051350
-116075026 59968278
-86232451 39534863
74812657 -87612262
43932309 -59781397
112280349 -84586419
-24836624 89582728
139050256 -48638499
139706961 -43229662
40721150 -49383022
-34204754 59049938
-123697687 98739078
40087318 -45110287
-25421041 80471756
62433483 1918193
126605914 -73777576
46967938 -65684258
41046337 -50936490
-74918480 138214409
-41196356 51206514
-66566077 38900137
-122963501 101705291
-26105468 99488324
-29843352 66367542
95676649 -89572059
-91297144 41008213
109953627 5849197
-109774237 52464622
-45807137 128937668
-82628814 137600638
39826095 -40878345
131267580 -11955280
-48718090 130861900
99446289 9147357
64924679 -83279969
-81241555 38628861
42543069 -23631959
48393741 -11914361
45945996 -15930192
121370676 -78702644
-124532638 93547880
-95701487 133644410
-80689791 38560499
-84334774 39128574
44357669 -60736403
49485943 -69470272
-25270855 94929439
138056360 -26765607
-79426657 138001651
126145643 -74270927
75028035 -87679483
139327033 -46898109
-98719701 132131522
-93420866 134624958
112234425 4776244
42182876 -55115469
-112315315 121289333
-116343258 116064910
-108603088 125073174
75854217 8094265
-42089946 126015053
78524794 8791589
-40739929 51626372
-30022491 66002548
41229135 -51716387
123554114 -76819148
-112979063 55907767
-124794639 86689521
-124120928 96533857
-111349100 54075509
-30104119 65838682
-41555958 125546041
-25402514 95838127
69113866 -85429109
49503842 -69494683
122101220 -1736532
-94801703 134047333
74565438 -87533787
115313666 3093985
48366657 -11954467
40546526 -31405700
-34442762 117707699
139713303 -36700506
81532535 9392307
-124676904 91966807
-83605744 137436191
-24820351 87719448
139816387 -40130444
-88287812 136365954
105091466 -87526345
110209510 -85568993
98706229 -89120063
106578108 7190281
-33415501 116247372
-122038150 71775287
45161344 -62408535
-122946241 74662286
-112144679 54947235
-44628037 128071406
103464039 8184878
114700194 3451790
-83753769 137409533
50728544 -71095214
-24866269 90413264
-116347130 116059134
126892373 -6369560
50270677 -9320857
-105336658 48608886
-24821248 88793078
-53499760 42986899
-52564241 43439801
-81981240 137698716
47500475 -66549773
61988175 1623303
-81508877 38664225
-124767376 85967318
