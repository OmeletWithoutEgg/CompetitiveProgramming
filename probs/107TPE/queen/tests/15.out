7460147.81 24155146.48
